src/projects/entities/project.entity.ts

```diff
@@ -0,0 +1,38 @@
+import mongoose, { Document } from 'mongoose';
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Address } from './address.entity';
+import { ProjectTasks } from './projectTasks.entity';
+
+enum StatusType {
+  START = 'start',
+  FINISHED = 'finished',
+  IN_PROGRESS = 'in-progress',
```

Comment:
Caso for serguir o padrão do "Task.entity", o ENUM lá ta em caps. Melhor deixar os dois ou tudo minusculo, ou  tudo maiúsculo (nos values).